app/tasks/maintenance_tasks/task.rb

```diff
@@ -72,6 +74,27 @@ def count
         new.count
       end
 
+      # Returns the list of the throttle specs for a given Task. This is
+      # provided as an array of hashes, with each hash specifying two keys:
+      # throttle_condition and backoff. Note that Tasks inherit throttle specs
+      # from their superclasses.
+
+      # @return [Array<Hash{Symbol => Proc, ActiveSupport::Duration}>]
+      def throttle_specs
```

Comment:
So this is a public API, in case people don't want to inherit? (They could use `throttle_specs.clear` to get rid of any inherited value).
Maybe we could avoid making it public, even though it would remove some possibilities?
If we keep it public I think we may want to think more about naming? `specs` is a bit fuzzy maybe?

Reply:
Ah you're right, don't think this needs to be public. I don't think being able to customize inheritance warrants making this public API - Task authors should better configure their parent Tasks / modules if they don't want certain throttling conditions inherited.